icons/export-icons-as-svg.py

```diff
@@ -12,9 +12,18 @@ def numToString(number):
 
 def makeSVG(pathString, width, height):
     return (
-        f'<svg xmlns="http://www.w3.org/2000/svg" '
-        f'viewBox="0 0 {width} {height}">'
-        f'<path d="{pathString}"/></svg>\n'
+        '<svg xmlns="http://www.w3.org/2000/svg" '
+        + f'viewBox="0 0 {width} {height}">'
+        + f'<path d="{pathString}"/></svg>\n'
+    )
+
+
+def makeCursorSVG(pathString, width, height):
+    return (
+        '<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" '
+        + f'viewBox="0 0 {width} {height}">'
+        + f'<path stroke="rgb(255,255,255)" stroke-width="55" d="{pathString}"/>'
```

Comment:
To be good web citizens, we should avoid outputting the path twice. This should work (untested):

```
<svg ... >
<defs><path id="icon-path" d="{pathString}"/></defs>
<use href="#icon-path" stroke="rgb(...)" stroke-width="55"/>
<use href="#icon-path"/>
</svg>
```

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs#example

Reply:
done